src/typescript.ts

```diff
@@ -16,24 +16,18 @@ function nameIsReservedKeyword(name: string): boolean {
 }
 
 /**
- * Will determine whether the tableName should be prefixed with the schemaName
+ * Returns a version of the name that can be used as a symbol name, e.g.
+ * 'number' --> 'number_'.
  */
-export function getTableTypeName(
```

Comment:
Looking at the code I realized this wasn't used for type names, just for symbol names. Sorry for the bad suggestion!